lib/squadster/accounts/user.ex

```diff
@@ -80,6 +82,15 @@ defmodule Squadster.Accounts.User do
     struct
     |> cast(filtered_params, @auth_fields)
     |> validate_required([:uid])
+    |> set_hash_id
+  end
+
+  def set_hash_id(%Ecto.Changeset{data: user} = changeset) do
+    unless user.hash_id do
```

Comment:
As far as I know, the usage of `unless` with `else` is bad manners in ruby. Apparently, there is better to use `case` or `if !` statement.

Reply:
It is not Ruby!!1 But you are right